### PR TITLE
[WIP] Transform code to es6

### PR DESCRIPTION
**Pull request by gberaudo**
This PR is about moving the code base to ES6 modules and switching to a webpack buildchain.
The Google Closure Compiler buildchain is abandonned (at least for now).
- [x] make Travis happy;
- [ ] rebase and reaply transform.

### CODE REVIEW

File: karma-conf.js [hunk 1]
```diff
@@ -7,7 +7,6 @@ var isDebug = process.argv.some(function(argument) {
 });
 
 module.exports = function(config) {
-  var closureLibPath = 'node_modules/google-closure-library/';
```
Comment (sbrunner) [inline karma-conf.js]: This will be replace by the webpack version ...

**Comment by ger-benjamin** [comment]
That's because of this kind of PR that Github stats are so useless :sweat_smile:

**Comment by gberaudo** [comment]
+12,023 −22,632
I am a bit surprised there are 10k lines gone.

File: package.json [hunk 2]
```diff
@@ -103,6 +103,7 @@
     "walk": "2.3.13",
     "webpack": "4.6.0",
     "webpack-cli": "2.0.15",
+    "webpack-dev-server": "^3.1.3",
```
Comment (sbrunner) [inline package.json]: Don't use `^`

**Pull request closed without merging**